src/Bands/Bands.js

```diff
@@ -2,17 +2,58 @@ import React, { Component } from 'react';
 import Band from '../Band/Band.js';
 
 export default class Bands extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: this.props.genreBands.filter(band => {
```

Comment:
You may run into problems with setting your initial state with the props that are being passed to this component - this is generally considered an anti-pattern in React since it will not update state when/if the prop changes... introducing bugs... and also making it so there isn't a 'single source of truth' for this data in the app.